packages/vuetify/src/composables/rtl.ts

```diff
@@ -0,0 +1,62 @@
+import { ref, inject, provide, computed } from 'vue'
+
+// Types
+import type { InjectionKey, Ref } from 'vue'
+import type { LocaleInstance } from './locale'
+
+export interface RtlOptions {
+  rtl: Record<string, boolean>
+  isRtl?: boolean
+}
+
+export interface RtlProps {
+  rtl?: boolean
+}
+
+export interface RtlInstance {
+  rtl: Record<string, boolean>
+  isRtl: Ref<boolean>
+}
+
+export const VuetifyRtlSymbol: InjectionKey<RtlInstance> = Symbol.for('vuetify:rtl')
+
+export function createRtl (localeScope: LocaleInstance, options?: RtlOptions) {
+  return createRtlScope({
+    rtl: options?.rtl ?? {}, // TODO: default rtl map
+    isRtl: ref(options?.isRtl ?? false),
+  }, localeScope)
+}
+
+export function createRtlScope (currentScope: RtlInstance, localeScope: LocaleInstance, options?: RtlProps): RtlInstance {
+  return {
+    isRtl: computed(() => {
+      if (!!options && typeof options.rtl === 'boolean') return options.rtl
```

Comment:
```suggestion
      if (options.rtl === true) return options.rtl
```

Reply:
this is a check against an undefined value. it should only use `options.rtl` if it has been set.